designation/software-engineer.md

```diff
@@ -0,0 +1,55 @@
+# Software Engineer
```

Comment:
Looks good @vishnuitta 

Can you push this under job-roles directory and it is good to merge.

Reply:
changed the directory Kiran.